battleship-server/src/main/test/java/com/com/epam/solid/nie/config/language/CommunicateProviderImplTest.java

```diff
@@ -0,0 +1,50 @@
+package com.epam.solid.nie.config.language;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+public class CommunicateProviderImplTest {
+    private CommunicateProviderImpl communicateProvider = new CommunicateProviderImpl();
+
+    @DataProvider
+    public static Object[][] languages() {
+        return new Object[][]{{Language.ENGLISH},
+                {Language.POLISH}
+        };
+    }
+
+    @Test(dataProvider = "languages")
+    public void populateShould_FillMapWithCommunicates(Language language) throws Exception {
+        communicateProvider.populate(language);
+        assertFalse(communicateProvider.isMapEmpty());
+    }
+
```

Comment:
If you want you could have one test instead of this two below if you add language to Data Provided